### DISCUSSION TITLE
"hold for menu" button is covered by skip button

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26705] [Ideas] [answered]
The "hold for menu" button cannot be clicked if the skip button is above it:

![image](https://github.com/ppy/osu/assets/35064754/2775fe29-cedd-4e48-bc74-9e9e6043e354)

I think if I hover the "hold for menu" button while the skip button is visible, the "hold for menu" button should take precedence because it's small and so the click is probably intentional and meant for the "hold for menu" button instead of the skip button. I can still reach the "hold for menu" button if the skip button is visible by a few pixels if I hover it at the very bottom of the button (this is the case in the screenshot above).

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/3535.
